### DISCUSSION TITLE
Suggestions for New Features Not Available Elsewhere

### DISCUSSION THREAD

**Discussion by ag346** [Klocman/Bulk-Crap-Uninstaller#315] [Ideas]
There are certain types of information that are available in the windows explorer and/or the programs and features windows that are not available in BCU and would be useful. 

For example: Update Info Link lists the official URL where you can download the latest version. Support Link lets you know where to go for help regarding the program. The Comments field seems as if it could be edited by the end-user in order for the person to enter a few useful reminders. The install source would be useful if it could be edited by the end-user in order to point to a folder where the person stores all of their installers. The downloads folder is typically just the initial storage location for the installer.

Lastly, when you open the properties windows for a given program, there is a previous versions tab that in my experience is always empty. It would be nice to have a way of referencing previous versions in a software folder specified by the end-user as opposed to having to rely on the windows file history or restore points, which in my experience never seem to contain previous versions of software installers.

**Comment by Klocman** [comment]
I started an issue so this doesn't get forgotten. #323
